### DISCUSSION TITLE
Goordhow Xiisad

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#13] [Announcements]
Waxaan qaban doonna goordhow xiisad aan ugu talagalnay. extension development-ga sida manifest, chrome API Insha'Allah!

waan soo dhacineynaa xiliga aan ka dhigi karno dhamaan waa in la wada imada Insha'Allah!

**Comment by AishaKeyf** [comment]
OK

**Comment by Nasrah-muse** [comment]
Insha Allah

**Comment by hayle01** [comment]
Insha allah eng

**Comment by iftiiin** [comment]
Ok

**Comment by A-M-Ahmed** [comment]
insha allah
